app/ts/ReportingUI/components/Reporting.tsx

```diff
@@ -77,9 +129,10 @@ export const Reporting = ({ maybeAccountAddress }: ReportingProps) => {
 		const marketAddress = EthereumAddress.safeParse(marketAddressString.value.trim())
 		if (!marketAddress.success) throw new Error('market not defined')
 		const parsedMarketAddressString = addressString(marketAddress.value)
-		const newMarketData = await fetchMarket(account.value, parsedMarketAddressString)
+		const newMarketData = await fetchHotLoadingMarketData(account.value, parsedMarketAddressString)
+		console.log(jsonStringify(newMarketData))
```

Comment:
Forgotten logging code, or intentional?  How hard would it be to make it so a query string parameter can be supplied that sets a state variable on the page that turns debug logging on/off?  Instead of `console.log` we would need some global helper function like:

```ts
window.logLevel = 0
function log(level, message) {
	if (level > logLevel) return
	console.log(message)
}
function logInfo(message) {
	log(1, message)
}
```

```suggestion
```